frontend/src/app/main/ui/components/design_system.cljs

```diff
@@ -0,0 +1,18 @@
+(ns app.main.ui.components.design-system
+  (:require
+   [app.main.features :as features]
+   [app.main.ui.components.buttons.simple-button :as sb]
+   [app.main.ui.icons :as icons]
+   [rumext.v2 :as mf]))
+
+(mf/defc story-wrapper
+  {::mf/wrap-props false}
+  [{:keys [children]}]
+    [:.default children])
+
+(def ^export default #js {
+  :icons #js {
+    :IconAddRefactor (mf/fnc icon-add-refactor [] icons/add-refactor)
```

Comment:
Just aliasing `IconAddRefactor` to `icons/add-refactor` didn't work. Is there an easy way to do this?

Reply:
It should be react component react element?

Reply:
Also, consider indenting this file using standard clojure formating